server/controllers/AdminControllers.js

```diff
@@ -0,0 +1,31 @@
+import models from '../database/models';
+import helpers from '../helpers';
+
+const { Author } = models;
+const { responseMessage } = helpers;
+
+/**
+ * @class AdminControllers
+ */
+export default class AdminControllers {
```

Comment:
This should be called 'AuthorController' since it's performing operations on author, while the route where the addAuthor static function will be used should be called admin, since it's exclusively the admin's job to add authors.

Reply:
I understand your point, but I would need further clarifications from @KvNGCzA , to know if the process should flow this way.

admin route -----> AuthorController  or  admin route ------> AdminController  (addAuthor)
and same also for signup and sign in
auth route  -------> UserController or auth route -----------> AuthController  (signUp and signIn)

Reply:
Treated accordingly